library/core/src/main/java/com/google/android/exoplayer2/audio/AudioTrack.java

```diff
@@ -1439,6 +1439,8 @@ private static int getEncodingForMimeType(String mimeType) {
         return C.ENCODING_DTS;
       case MimeTypes.AUDIO_DTS_HD:
         return C.ENCODING_DTS_HD;
+      case MimeTypes.AUDIO_TRUEHD:
```

Comment:
Around L588 do we need to set a buffer size for TrueHD output or is the buffer size for DTS fine to use? (In the latter case, please update the comment to refer to TrueHD in addition to DTS and DTS HD.)

Reply:
Adding to the comment as any truehd tracks I've played seem fine with the DTS buffer size.  On a side note I have another pull request for a buffer size change because in 7.1 tracks I get audio dropouts on DTS-HD MA tracks.  5.1 tracks seem fine.